From de6e640f6febdc263de6ed4d9fee65d7a5bb00db Mon Sep 17 00:00:00 2001
From: Gunnar Mills <gmills@us.ibm.com>
Date: Thu, 27 Sep 2018 08:53:07 -0500
Subject: [PATCH 1/1] Set the Host Time if "Split" and "NTP"

From: https://github.com/openbmc/phosphor-time-manager
Mode	Owner	Set BMC Time	Set Host Time
NTP	SPLIT	Fail to set	OK

Resolves openbmc/phosphor-webui#19

Change-Id: I72f5617581533a2bb17d415b45bdf7ec6372ecab
Signed-off-by: Gunnar Mills <gmills@us.ibm.com>
---
 app/configuration/controllers/date-time-controller.js | 8 ++++++--
 1 file changed, 6 insertions(+), 2 deletions(-)

diff --git a/app/configuration/controllers/date-time-controller.js b/app/configuration/controllers/date-time-controller.js
index 5a1f8d2..c5a3e8b 100644
--- a/app/configuration/controllers/date-time-controller.js
+++ b/app/configuration/controllers/date-time-controller.js
@@ -101,11 +101,15 @@ window.angular && (function(angular) {
                   manual_promises.push(setBMCTime());
                 }
 
-                if ($scope.time_owner == 'Host' ||
-                    $scope.time_owner == 'Split') {
+                if ($scope.time_owner == 'Host') {
                   manual_promises.push(setHostTime());
                 }
               }
+              // Set the Host if Split even if NTP.
+              if ($scope.time_owner == 'Split') {
+                manual_promises.push(setHostTime());
+              }
+
               $q.all(manual_promises)
                   .then(
                       function() {
-- 
1.8.3.1

